frontend/src/App.tsx

```diff
@@ -1,24 +1,17 @@
 import './App.css';
 import Event from './Event/Event';
 import bbq from './assets/bbq.png';
+import Society from './Society/Society';
 
 function App() {
   return (
     <>
-      <h1>Hi</h1>
-      <Event
-        name="DevSoc BBQ"
-        image={bbq}
-        backgroundPositionY="180px"
-        time="Tomorrow mate"
-      ></Event>
-      <Event
-        name="TikTok rizz party"
-        image="https://pbs.twimg.com/media/F2y8Ehbb0AA-ch9.jpg"
-        backgroundPositionY="150px"
-        time="25th Dec 2024"
-        keywords={['hiii', 'another keyword']}
-      ></Event>
+      <Society 
+      name="RizzSoc"
+      image="https://media.tenor.com/c6_k3koMQckAAAAe/rizz-flower.png" 
+      backgroundPositionY='150px'
+      color='#F3E1E1'    
```

Comment:
i would suggest using [hsl](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl) instead of hex - this is just personal preference but it would be good to see it throughout the project